packages/blade/src/components/Box/Box.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import type { Meta } from '@storybook/react';
+import React from 'react';
+import { getBoxArgTypes } from './BaseBox/storybookArgTypes';
+import type { BoxProps } from '.';
+import { Box } from '.';
+import { Text, Title } from '~components/Typography';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+import { LinkToStorybook } from '~src/_helpers/storybook/LinkToStorybook';
+
+if (window.top) {
+  document.getElementById(window.top.location.hash)?.scrollIntoView();
+}
+
+const BoxStoryMeta = {
+  title: 'Components/Layout Primitives (Box)/Box',
+  component: Box,
+  argTypes: getBoxArgTypes(),
+  parameters: {
+    docs: {
+      page: () => (
+        <StoryPageWrapper
+          componentName="Box"
+          componentDescription="Box Component from Layout Primitives of Blade."
+          propsDescription="All Box props support responsive objects. Props marked with 💅🏼 next to their names are the props that can also be used as styled-props on other blade components. Check out styled-props documentation for more details."
```

Comment:
can we add a table group and group all the properties that are common to Box and exposed as style props so its easier to differentiate?

Reply:
by table group you mean category? that we've added for other components to differentiate between their own props vs styled-props. In Box, we technically don't have styled-props. They are just box props.

I have marked it here to show consumers that this prop can 'also' be used as style-props but I feel having 2 different groups would get confusing because then padding will be boxProp but margin will be in styledProps section.